src/core/lib/pick-layers.js

```diff
@@ -18,192 +18,40 @@
-        });
-      }
-
-      withParameters(gl, {viewport: glViewport}, () => {
-
-        layer.drawLayer({
-          moduleParameters: Object.assign({}, layer.props, {
-            viewport: layer.context.viewport
-          }),
-          uniforms: Object.assign(
-            // TODO: Update all layers to use 'picking_uActive' (picking shader module)
-            // and then remove 'renderPickingBuffer' and 'pickingEnabled'.
-            {
-              renderPickingBuffer: 0,
-              pickingEnabled: 0,
-              picking_uActive: 0
-            },
-            layer.context.uniforms,
-            {layerIndex}
-          ),
-          parameters: layer.props.parameters || {}
-        });
-
-      });
-
-      visibleCount++;
-    }
-  });
-  const totalCount = layers.length;
-  const primitiveCount = totalCount - compositeCount;
-  const hiddenCount = primitiveCount - visibleCount;
-
-  const message = `\
-#${renderCount++}: Rendering ${visibleCount} of ${totalCount} layers ${pass} \
-(${hiddenCount} hidden, ${compositeCount} composite)`;
-
-  log.log(2, message);
-}
-
-// Pick all objects within the given bounding box
-export function queryLayers(gl, {
-  layers,
-  pickingFBO,
-  x,
-  y,
-  width,
-  height,
-  viewport,
-  mode,
-  useDevicePixelRatio
-}) {
-
-  // Convert from canvas top-left to WebGL bottom-left coordinates
-  // And compensate for pixelRatio
-  const pixelRatio = getPixelRatio({useDevicePixelRatio});
-  const glViewport = getGLViewport(gl, {viewport, pixelRatio});
-
-  const deviceLeft = Math.round(x * pixelRatio);
-  const deviceBottom = Math.round(gl.canvas.height - y * pixelRatio);
-  const deviceRight = Math.round((x + width) * pixelRatio);
-  const deviceTop = Math.round(gl.canvas.height - (y + height) * pixelRatio);
-
-  // Only return unique infos, identified by info.object
-  const uniqueInfos = new Map();
-
-  withParameters(gl, {viewport: glViewport}, () => {
-
-    const pickInfos = getUniquesFromPickingBuffer(gl, {
-      layers,
-      pickingFBO,
-      deviceRect: {
-        x: deviceLeft,
-        y: deviceTop,
-        width: deviceRight - deviceLeft,
-        height: deviceBottom - deviceTop
-      }
-    });
-
-    pickInfos.forEach(pickInfo => {
-      let info = createInfo([pickInfo.x / pixelRatio, pickInfo.y / pixelRatio], viewport);
-      info.devicePixel = [pickInfo.x, pickInfo.y];
-      info.pixelRatio = pixelRatio;
-      info.color = pickInfo.pickedColor;
-      info.index = pickInfo.pickedObjectIndex;
-      info.picked = true;
-
-      info = getLayerPickingInfo({layer: pickInfo.pickedLayer, info, mode});
-      if (!uniqueInfos.has(info.object)) {
-        uniqueInfos.set(info.object, info);
-      }
-    });
-  });
-
-  return Array.from(uniqueInfos.values());
-}
 
 /* eslint-disable max-depth, max-statements */
 // Pick the closest object at the given (x,y) coordinate
-export function pickLayers(gl, {
+export function pickObject(gl, {
```

Comment:
This is a name change to a public method, is the documentation change planned for next PR?

Reply:
> This is a name change to a public method, is the documentation change planned for next PR?

It is exported from this file for use in other files, but it is not documented, and as far as I know it is not exposed from index.js (which is in a sense the "source of truth" for what is considered public).